test.sh

```diff
@@ -75,13 +75,28 @@ if [[ $dockerId =~ "ldc" ]] ; then
     source=$'void main() { \n int a; \n int* ap = &a + 1; \n *ap = 0; }'
     #                                                        ^ line 4 column 2
     bsource=$(echo "$source" | base64 -w0)
-    DOCKER_FLAGS="-fsanitize=address -g -disable-fp-elim" docker run -e DOCKER_FLAGS --rm $dockerId $bsource | grep -q '#0 0x[0-9a-f]* in ..main [/a-z\.]*:4:2'
+    DOCKER_FLAGS="-fsanitize=address -g" docker run -e DOCKER_FLAGS --rm $dockerId $bsource | grep -q '#0 0x[0-9a-f]* in ..main [/a-z\.]*:4:2'
```

Comment:
`-disable-fp-elim` was [removed in 1.24.0-beta1](https://github.com/ldc-developers/ldc/issues/3576)